File: WebAPI/WebAPI.Models/Basic/Sprint.cs
using System;

namespace WebAPI.Models.Basic
{
    public class Sprint
    {
        public Guid SprintId { get; set; }

        public Guid EpicId { get; set; }
        
        public string SprintName { get; set; }
        
        public DateTime StartDate { get; set; }
        
        public DateTime EndDate { get; set; }
        
        public DateTime CreationDate { get; set; }
        
        public bool IsDeleted { get; set; }
    }
}

File: web/src/mappers/sprint.ts
import { IFullSprint, ISprint } from '../types/sprint';

export const mapToSprintModel = (data): ISprint => ({
    sprintId: data.sprintId,
    epicId: data.epicId,
    sprintName: data.sprintName,
    startDate: data.startDate,
    endDate: data.endDate,
    creationDate: data.creationDate,
});

export const mapFullSprintToSprint = (fullSprint: IFullSprint): ISprint => ({
    sprintId: fullSprint.sprintId,
    epicId: fullSprint.epicId,
    sprintName: fullSprint.sprintName,
    startDate: fullSprint.startDate,
    endDate: fullSprint.endDate,
    creationDate: fullSprint.creationDate,
});
